acceptance/tests/restart_host_run_puppet.rb

```diff
@@ -15,7 +15,16 @@
   applicable_agents = applicable_agents.select { |agent| agent['platform'] != "eos-4-i386"}
   unless applicable_agents.length > 0 then
     skip_test('RE-7673 - Arista hosts cannot be restarted')
-  end  
+  end
+
+  # QENG-4371 The fedora hosts in our VM Pooler consistently take 10+ minutes
+  # to reboot, causing this test to fail sporadically (~20% of the time). Once
+  # QENG-4371 is resolved, this platform should be returned to testing.
+  applicable_agents = applicable_agents.select { |agent| agent['platform'] !~ /fedora/ }
```

Comment:
I noticed that when testing PCP-585 this week, fedora24-32a returns from a reboot in timely manner; so it is not all Fedora instances. 
However, PCP-585 is another issue with the same test on that platform, so it might be simplest to skip all Fedora variants for now?